src/scripts/main.js

```diff
@@ -1,7 +1,45 @@
 'use strict';
 
+const body = document.body;
+
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const div = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  h2.textContent = title;
+  p.textContent = description;
+
+  div.classList.add('notification');
+  h2.classList.add('title');
+
+  div.prepend(h2, p);
+  body.append(div);
+
+  const computedStyle = window.getComputedStyle(div);
+  const width = parseInt(computedStyle.width);
+  const padding =
+    parseInt(computedStyle.paddingLeft) + parseInt(computedStyle.paddingRight);
+
+  div.style.minWidth = `${width + padding}px`;
+  div.style.right = `${posRight}px`;
+  div.style.top = `${posTop}px`;
+
+  switch (type) {
+    case 'success':
+      div.classList.add('success');
+      break;
+
+    case 'error':
+      div.classList.add('error');
+      break;
+
+    case 'warning':
+      div.classList.add('warning');
+  }
```

Comment:
There is a missing 'break' statement in the 'warning' case of the switch block. Although not strictly necessary here, it's a good practice to include it to prevent potential fall-through issues if more cases are added in the future.